dependencies.gradle

```diff
@@ -49,7 +49,7 @@ ext.dep = [
   "logbackClassic": "ch.qos.logback:logback-classic:1.2.3",
   "logbackJsonCore": "ch.qos.logback.contrib:logback-json-core:0.1.5",
   "loggingApi": "io.github.microutils:kotlin-logging:1.4.9",
-  "miskWeb": "com.squareup.misk-web:misk-web:2018.11.20-06a2d00",
+  "miskWeb": "com.squareup.misk-web:misk-web:2018.11.24-8047665",
```

Comment:
How come we need a gradle dependency if it's all js stuff?

Reply:
So the Jar ships with the necessary common JS code.

Reply:
I thought misk-web was just stuff from npm?